README.md

```diff
@@ -266,21 +265,18 @@ Enterprise](https://konghq.com/kong-enterprise-edition/).
 
 ## License
 
-```
+
 Copyright 2016-2020 Kong Inc.
 
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
It is redirecting to license

Reply:
LICENSE text should not be edited. Otherwise it is not that LICENSE anymore.